caravel/assets/package.json

```diff
@@ -38,7 +38,6 @@
     "autobind-decorator": "^1.3.3",
     "bootstrap": "^3.3.6",
     "bootstrap-datepicker": "^1.6.0",
-    "bootstrap-toggle": "^2.2.1",
```

Comment:
i replaced the toggles in expore.jsx with simple checkboxes, but maybe i missed an instance?